library/spdm_common_lib/libspdm_com_context_data_session.c

```diff
@@ -19,6 +19,32 @@ void libspdm_session_info_init(libspdm_context_t *spdm_context,
     libspdm_session_type_t session_type;
     uint32_t capabilities_flag;
 
+    if (session_id != INVALID_SESSION_ID) {
+        if (use_psk) {
+            if (spdm_context->max_psk_session_count != 0) {
```

Comment:
This might be picked up by a security scan / compiler warning if the `LIBSPDM_ASSERT` isn't present, as it's just an empty `if` statement.

Reply:
done